contrib/inventory/ec2.py

```diff
@@ -647,6 +647,8 @@ def get_rds_instances_by_region(self, region):
                         instance.tags = {}
                         for tag in tags:
                             instance.tags[tag['Key']] = tag['Value']
+                        # collect the VPC ID from the DB subnet group
+                        instance.vpc_id = db_instances['DBInstances'][index]['DBSubnetGroup']['VpcId']
```

Comment:
This doesn't seem to do anything - it adds the information but then doesn't use it for grouping or display, as far as I can tell. Am I missing something?